src/pages/Main/PetProductBox.js

```diff
@@ -1,28 +1,47 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import "./PetProductBox.scss";
+import { Link } from "react-router-dom";
 
 export default function PetProductBox() {
   const [index, setIndex] = useState(1);
+  const [color, setColor] = useState("#fccf1d");
+  const changeColor = () => {
+    if (index === 1) {
+      setColor("#fccf1d");
+    } else if (index === 2) {
+      setColor("#c81a20");
+    } else if (index === 3) {
+      setColor("#016ad5");
+    } else if (index === 4) {
+      setColor("#cda5e0");
+    } else if (index === 5) {
+      setColor("#d8e22d");
+    }
+  };
+  useEffect(() => {
+    changeColor();
+  });
```

Comment:
```suggestion
  useEffect(() => {
    changeColor();
  }, [] );
```

빈배열이나 디펜던시 없어도 되는 코드인가요?

Reply:
이거 일부러 계속 사용하려고 저렇게 한 건데 빈 배열로 해보니까 색깔이 아예 안바껴서 일단 이렇게 해놓고 시간있을 때 디펜던시를 찾아 넣을 계획입니다